.github/actions/regression-tests/action.yml

```diff
@@ -73,11 +87,11 @@ runs:
         TIMEDOUT_STEP_1: ${{ steps.first.outputs.TIMEDOUT }}
         TIMEDOUT_STEP_2: ${{ steps.second.outputs.TIMEDOUT }}
       run: |
-        if [[ "${{ env.TIMEDOUT_STEP_1 }}" -eq 1 ]] || [[ "${{ env.TIMEDOUT_STEP_2 }}" -eq 1 ]]; then
           echo "🪵🪵🪵🪵🪵🪵 Latest log before timeout 🪵🪵🪵🪵🪵🪵\n\n"
```

Comment:
These will be removed on a separate PR.

1. Will rename all the logs generated to correspond to the id of the instance we assign in instance.py
2. Will only upload the logs of failed tests